app/code/Magento/GiftMessage/Block/Cart/GiftOptions.php

```diff
@@ -78,8 +78,8 @@ public function getJsLayout()
      *
      * @return array
      */
-    public function getGiftOptionsConfigJson()
+    public function getGiftOptionsConfig()
```

Comment:
Hi, @jonathanKingston , due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't rename public methods. 
Please create the new method.